packages/flutter/lib/src/widgets/editable.dart

```diff
@@ -103,6 +126,10 @@ class EditableString {
   /// The current text being edited.
   String get text => _client.text;
 
+  void setText(String text) {
+    _client.text = text;
+  }
```

Comment:
I believe this via the _keyboardHandle.setEditingState(_editableString.editingState) calls in Input